docs/user/alternatives.rst

```diff
@@ -67,18 +67,14 @@ These are metrics that can be scraped from GitHub.
 
 **Stars**: The number of people who have starred the repository. This is not the best metric for number of users, and instead gives you a better idea about the amount of *hype* surrounding the project.
 
-**Issues**: The total number of open and closed issues. Although it may seem counterintuitive, the more issues, the better. Large projects like PyTorch have tens of thousands of open issues. This does not mean that PyTorch is broken, it means that it is popular and has enough issues to discover corner cases or open feature requests.
+**Issues**: The total number of open issues. Although it may seem counterintuitive, the more issues, the better. Large projects like PyTorch have tens of thousands of open issues. This does not mean that PyTorch is broken, it means that it is popular and has enough issues to discover corner cases or open feature requests.
 
 **PRs**: The total number of open and closed pull requests. This tells you how active development of the project has been. Note that this metric can be artificially inflated by bots like dependabot.
 
 **Releases**: The number of software releases. The frequency of releases varies from project to project. The important thing to look for is multiple releases.
 
 **Commits**: The number of commits on the main development branch. This is another metric for how active development has been. However, this can vary a lot depending on whether PRs are merged with or without squashing first.
 
-**Core SLOCs**: The number of source lines of code in the core library, excluding empty lines and comments. This tells you how large the library is, and how long it would take someone to write something like it themselves. We use `scc <https://github.com/boyter/scc>`_ to compute SLOCs and exclude markup languages from the count.
-
-**Test SLOCs**: The number of source lines of code in the testing suite, excluding empty lines and comments. This tells you how well tested the project is. A good goal to strive for is a similar amount of code for testing as there is in the core library itself.
```

Comment:
No way to get SLOCs from GitHub. Could do a shallow clone of every repo, run scc on specific hard-coded directories, parse the output back into Python, and filter only programming languages, but this feels more complicated and slow than it's worth.